p2pserver/peer/config.json

```diff
@@ -0,0 +1,30 @@
+﻿
```

Comment:
Duplicated. Please remove it.

Reply:
After default config fix, the file is useless . I will remove it